### DISCUSSION TITLE
How to configure RaspAP with two external wifi adapters?

### DISCUSSION THREAD

**Discussion by paolosezart** [RaspAP/raspap-webgui#1691] [Q&A] [answered]
I have a raspberry pi 4b. RaspAP version 3.1.9
I am trying to set up a RaspAP with two external wifi adapters. 
Clients can connect to the RaspAP, but there is no internet, neither on the raspberry pi nor on the clients.
Can you please advise me what I need to configure to make it work?

<details> 
  <summary>Here are the screenshots:</summary>
 
![0](https://github.com/user-attachments/assets/6da36d6d-5a6b-401d-9135-1b57d80a4de3)
![1](https://github.com/user-attachments/assets/91d01b10-2cf8-4f75-8ddd-51be6307ec92)
![2](https://github.com/user-attachments/assets/41ab311f-d4b7-4e52-9299-a445f17c517a)
![3](https://github.com/user-attachments/assets/40dfbd8b-911d-4d5e-9da2-41a783dd8a6b)
![4](https://github.com/user-attachments/assets/4adef428-94f8-418d-89da-024984ab5e0a)

</details>

**Comment by billz** [comment, marked as answer]
Follow the [configure routing steps](https://docs.raspap.com/repeater/#configure-routing), substituting your interfaces (wlan1 and wlan2) for the ones mentioned in the docs.
